### PR TITLE
feat: add Changelog page

### PR DESCRIPTION
**Pull request by vannyle**
**Describe** what changes this pull request brings
This pull request adds Changelog page.

**Steps to test**
1. Open the [page](https://websitemain62807-changelog.gtsb.io/docs/release-notes/)
2. Confirm that everything looks and works as expected

<details>
<summary>Screenshots</summary>

 Changelog page
Screenshot 2022-08-01 at 17 45 49

 Changelog page pagination
Screenshot 2022-08-01 at 17 46 22

 Changelog post
Screenshot 2022-08-01 at 17 46 29

</details>

**References**
[Design](https://www.figma.com/file/6WyZRL33uvZsBewEWxB3Si/Neon-Website-Design?node-id=6108%3A15447)

### CODE REVIEW

File: gatsby/create-changelog-posts.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          }
+        ) {
+          nodes {
+            id
+            slug
+            fields {
+              redirectFrom
+            }
+            frontmatter {
+              title
```
Comment (bolotskydev) [inline gatsby/create-changelog-posts.js:27]: @vannyle it seems we don't really need `redirectFrom` and `title` fields at changelog entries

File: gatsby/create-pages.js [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
   await createDocPages(options);
+  await createChangelogPages(options);
+  await createChangelogPosts(options);
+
+  const { createRedirect } = options.actions;
+
+  createRedirect({
+    fromPath: `/docs/release-notes/`,
+    toPath: `/changelog/`,
+  });
```
Comment (bolotskydev) [inline gatsby/create-pages.js:22]: This one is redundant since we introduced `standalone` flag? 🤔

File: gatsby/create-changelog-posts.js (hunk 1 above)
Comment (vannyle) [reply gatsby/create-changelog-posts.js]: Removed `redirectFrom` and `title`

**Review by danieltprice — COMMENTED** [review]
I like the new look. A few thoughts: 

-  I think the heading should be "Release Notes" to match the navigation link from the Docs page. 
- With that change, I suggest keeping the current URL (https://neon.tech/docs/release-notes/).
- I also suggest changing "Latest product updates by Neon’s team" to something like, "The latest product updates from Neon", or "The latest features, enhancements, and fixes from Neon".

**Review by danieltprice — APPROVED** [review]

**Review by stepashka — DISMISSED** [review]

**Review by bolotskydev — APPROVED** [review]

**Pull request merged**